vendor/github.com/coreos/go-oidc/oidc/provider.go

```diff
@@ -353,9 +353,6 @@ func (p ProviderConfig) Valid() error {
 	if !contains(p.IDTokenSigningAlgValues, "RS256") {
 		return errors.New("id_token_signing_alg_values_supported must include 'RS256'")
 	}
-	if contains(p.TokenEndpointAuthMethodsSupported, "none") {
```

Comment:
we've seen instances of misconfigured IDPs that included this algorithm... is there a good reason we would allow it?